skyportal/handlers/api/group_admission_request.py

```diff
@@ -86,7 +86,7 @@ def get(self, admission_request_id=None):
         self.verify_and_commit()
         return self.success(data=response_data)
 
-    @auth_or_token
+    @permissions(["Upload data"])
```

Comment:
Should still allow even the view-only users to request admission to groups?

Reply:
Sounds reasonable -- updated.